content/docs/command-reference/unfreeze.md

```diff
@@ -40,51 +40,38 @@ First, let's create a simple DVC-file:
 ```dvc
 $ echo foo > foo
 $ dvc add foo
-Saving information ...
-
-$ dvc run -d foo -o bar cp foo bar
-Running command:
-  cp foo bar
+$ dvc run -n make_copy -d foo -o bar cp foo bar
 ...
 ```
 
-Then, let's change the file `foo` that the stage described in `bar.dvc` depends
-on:
+Then, let's change the file `foo` that the stage `make_copy` depends on:
 
 ```dvc
 $ rm foo
 $ echo foo1 > foo
 $ dvc status
 
-bar.dvc
-        deps
-                changed:  foo
-foo.dvc
-        outs
-                changed:  foo
+make_copy:
+	changed deps:
+		modified:           foo
+foo.dvc:
+	changed outs:
+		modified:           foo
 ```
 
-Now, let's freeze the `bar` stage:
+Now, let's freeze the `make_copy` stage:
 
 ```dvc
-$ dvc freeze bar.dvc
+$ dvc freeze make_copy
 $ dvc status
 
-  foo.dvc
-          outs
-                  changed:  foo
+foo.dvc:
+	changed outs:
+		modified:           foo
 ```
 
 Run `dvc unfreeze` to unfreeze it back:
 
 ```dvc
-$ dvc unfreeze bar.dvc
-$ dvc status
-
-  bar.dvc
-          deps
-                  changed:  foo
-  foo.dvc
-          outs
-                  changed:  foo
+$ dvc unfreeze make_copy
```

Comment:
https://github.com/iterative/dvc.org/pull/1488/files#r444374258

I think there is no need to put the statement in the end, as it is stated in the beginning:
"Run `dvc unfreeze` to unfreeze it back."